src/main/kotlin/no/nav/k9/jobber/RekjøreGrafer.kt

```diff
@@ -44,14 +44,8 @@ fun Application.rekjørEventerForGrafer(
             val alleEventerIder = behandlingProsessEventK9Repository.hentAlleEventerIder()
```

Comment:
Dette er bare refaktorering, eller bruker dere denne rekjørEventerForGrafer?

Reply:
Bare refaktorering. Logging løy om ting som var gjort (logger for tidlig), og mapping til og fra UUID var et gå-over-bekken-etter-vann tilfelle.

Reply:
👍  - jeg tror ikke rekjørEventerForGrafer brukes i dag, mulig den er ment for manuell rekjøring, men vi har aldri brukt den :D